stdlib/public/stubs/Stubs.cpp

```diff
@@ -252,8 +258,43 @@ extern "C" uint64_t swift_float80ToString(char *Buffer, size_t BufferLength,
 /// if an error occurred, or EOF was reached.
 SWIFT_RUNTIME_STDLIB_INTERFACE
 extern "C" ssize_t swift_stdlib_readLine_stdin(char **LinePtr) {
+#if defined(_MSC_VER)
```

Comment:
I still think that the first thing this function should do, for all platforms, is: *LinePtr = NULL.
It's an out parameter, according to the docstring, not in/out.